src/modm/board/hifive1/board.hpp

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2019 Benjamin Weps
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef MODM_HIFIVE1_HPP
+#define MODM_HIFIVE1_HPP
+
+#include <modm/platform.hpp>
+
+#include <cstdint>
+
```

Comment:
```suggestion
using namespace modm::platform;
```
And then remove the `modm::platform::*` in the BSP;